File: data/deepseek_coder_base_6.7B/readable_format_HPEC/fortran/file076.f90
program DRB003_antidep2_orig_yes
    use omp_lib
    implicit none

    integer :: i, j, len
    real :: a(20,20)

    len = 20

    do i = 1, len
        do j = 1, len
            a(i,j) = 0.5
        end do
    end do

    !$omp parallel do private(j)
    do i = 1, len-1
        do j = 1, len
            a(i,j) = a(i,j) + a(i+1,j)
        end do
    end do
    !$omp end parallel do

    write(*,*) 'a(10,10) =', a(10,10)

end program

File: data/gpt-4o/readable_format_HPEC/cpp/file076.cpp
#include <stdio.h>
float a[20][20];
int main()
{
 int i,j;
 for (i=0;i<20;i++)
 for (j=0;j<20;j++)
 a[i][j]=0.5;
#pragma omp parallel for private(j)
 for (i=0;i<19;i++)
 for (j=0;j<20;j++)
 a[i][j]=a[i][j]+a[i+1][j];
 printf ("a[10][10]=%f
", a[10][10]);
 return 0;
}
